pyatv/__main__.py

```diff
@@ -158,6 +158,15 @@ def _handle_command(args, loop):
             _print_commands('Metadata', metadata)
             _print_commands('Playing commands', playing, newline=False)
 
+        elif cmd == 'artwork':
+            artwork = yield from atv.metadata.artwork()
+            if artwork is not None:
+                file = open('artwork.png', 'wb')
```

Comment:
I would recommend to use the ``with`` statement for opening tile file as that closes it no matter what happens.